README.md

```diff
@@ -1,19 +1,24 @@
-tree-sitter-scala
-=================
+# tree-sitter-scala
```

Comment:
My neovim did this change totally on his own, no idea why. Maybe it is not a recommended md syntax anymore?
I can revert it if you want.